src/routing/RouteSelector.jsx

```diff
@@ -32,7 +32,7 @@ export const RouteSelector = () => (
       <Banner variant={BANNER_VARIANT}>{BANNER_MESSAGE}</Banner>
     )}
     <Switch>
-      <RedirectIfLoggedInRoute exact path="/" component={Home} />
+      <RedirectIfLoggedInRoute exact path="/" component={LoginPage} />
```

Comment:
this is p confusing at a glance because i'd expect this to redirect to `LoginPage` if logged in ._.

also, as this component is only used here, i'm in favour of just deleting it and replacing it with a `Route` + `Redirect`

Reply:
The login page is only shown if you're not logged in, so this component redirects you to `/sites` if you're already logged in. I think it's neater if separated though!

Reply:
sure, could we then rename the prop from `component ` to something more obvious like perhaps `unauthComponent` so that the confusion on whether the component is shown if it's a successful login or not is reduced

Reply:
Sure, 43792f09